docs/content/preview/drivers-orms/rust/diesel.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Diesel ORM
+linkTitle: Diesel ORM
+description: Diesel ORM support for YugabyteDB
+headcontent: Diesel ORM support for YugabyteDB
+image: /images/section_icons/sample-data/s_s1-sampledata-3x.png
+menu:
+  preview:
+    identifier: diesel-orm
+    parent: rust-drivers
+    weight: 600
+isTocNested: true
+showAsideToc: true
+---
+[Diesel](https://diesel.rs/) is a safe, extensible object-relational mapping (ORM) tool and a query builder for Rust. It is the most productive way to interact with databases in Rust because of its safe and composable abstractions over queries. It eliminates the possibility of incorrect database interactions at compile time. It is designed to be abstracted over. It enables programmers to write reusable code and think in terms of your problem domain.
+
+YugabyteDB YSQL API has full compatibility with Diesel ORM for Data persistence in Rust applications. This page provides details for getting started with Diesel ORM for connecting to YugabyteDB.
+
+## Create a New Rust-Diesel Project 
+
+1. First, Ensure you have Rust (1.58.1) and Cargo(1.58.0) installed. If not installed. 
+2. Create a new project by using this:
+
+```shell
+$ cargo new --lib diesel_demo && cd diesel_demo
```

Comment:
This command doesn't succeed on my macOS 12.3 machine WITH libpq installed. I believe there's at least one environment variable that we need to be setting here.